testsuite/cluster-test/src/experiments/load_test.rs

```diff
@@ -315,14 +369,54 @@ async fn state_sync_load_test(
             ) = payload
             {
                 // TODO analyze response and update StateSyncResult with stats accordingly
-                served_txns += chunk_response.txn_list_with_proof.transactions.len();
+                served_txns += chunk_response.txn_list_with_proof.transactions.len() as u64;
             }
         }
     }
-    Ok(StateSyncResult { served_txns })
+    Ok(StateSyncStats {
+        served_txns,
+        bytes,
+    })
 }
 
-// TODO store more stats here
-struct StateSyncResult {
-    pub served_txns: usize,
+#[derive(Debug, Default)]
+struct StateSyncStats {
+    served_txns: u64,
```

Comment:
might also be good to get actual number of state sync msgs exchanged (same for mempool) - can either keep track here or get from prometheus counters